Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -134,106 +127,38 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _View, UIConfigP
-        ]
-
-        // Subtitle label
-        layout.subtitleLabelConstraints = [
-            // Top of the label is aligned with avatar vertical center
-            subtitleLabel.topAnchor.pin(equalTo: visualCenterGuide.bottomAnchor),
-            
-            // Pin the subtitle label leading anchor to avatar's trailing + spacing
-            subtitleLabel.leadingAnchor.pin(equalToSystemSpacingAfter: avatarView.trailingAnchor)
-        ]
-
-        // Unread count view
-        layout.unreadCountViewConstraints = [
-            // Pin the label to the trailing anchor
-            unreadCountView.trailingAnchor.pin(equalTo: layoutMarginsGuide.trailingAnchor),
-            
-            // Align it vertically with the title
-            unreadCountView.centerYAnchor.pin(equalTo: titleLabel.centerYAnchor),
-            
-            // Title label shouldn't overlap
-            unreadCountView.leadingAnchor.pin(greaterThanOrEqualToSystemSpacingAfter: titleLabel.trailingAnchor)
-        ]
 
-        // Set titleLabel compression resistance smaller than the unread count view
-        titleLabel.setContentCompressionResistancePriority(.streamLow, for: .horizontal)
-        unreadCountView.setContentCompressionResistancePriority(.streamRequire, for: .horizontal)
-
-        // Timestamp label
-        layout.timestampLabelConstraints = [
-            // Pin the label to the trailing anchor
-            timestampLabel.trailingAnchor.pin(equalTo: layoutMarginsGuide.trailingAnchor),
-            
-            // Align it vertically with the subtitle
-            timestampLabel.centerYAnchor.pin(equalTo: subtitleLabel.centerYAnchor),
-            
-            // Subtitle label shouldn't overlap
-            timestampLabel.leadingAnchor.pin(greaterThanOrEqualToSystemSpacingAfter: subtitleLabel.trailingAnchor)
-        ]
+        bottomContainer.addArrangedSubviews([
+            subtitleLabel.flexible(axis: .horizontal), timestampLabel
+        ])
 
-        // Set subtitleLabel compression resistance smaller than the timestamp label
-        subtitleLabel.setContentCompressionResistancePriority(.streamLow, for: .horizontal)
-        timestampLabel.setContentCompressionResistancePriority(.streamRequire, for: .horizontal)
+        NSLayoutConstraint.activate([
+            avatarView.heightAnchor.pin(equalToConstant: 48),
+            avatarView.widthAnchor.pin(equalTo: avatarView.heightAnchor)
+        ])
 
-        NSLayoutConstraint.activate(
-            layout.avatarViewConstraints
-                + layout.timestampLabelConstraints
-                + layout.titleLabelConstraints
-                + layout.subtitleLabelConstraints
-                + layout.unreadCountViewConstraints
-        )
+        mainContainer.addArrangedSubviews([
+            avatarView, ContainerView(axis: .vertical, spacing: 4, views: [topContainer, bottomContainer])
+        ])
+        
+        mainContainer.alignment = .center
```

Comment:
Is there a reason why this is not specified in its initializer? If user override `mainContainer` and initialize it with `ContainerView(alignment: .axisLeading)` then it will be overridden by this which might be a little bit counterintuitive.

Reply:
@fortmarek  There's no other reason than having all layout code in one place. I think you could say the same thing about all other elements :) If you override the variable and provide some custom values there, it will be overridden by customization methods...